resources/test/fixtures/pydocstyle/D401.py

```diff
@@ -0,0 +1,31 @@
+# Bad examples
+
+def bad_liouiwnlkjl():
+    """Returns foo."""
+
+
+def bad_sdgfsdg23245():
+    """Constructor for a foo."""
+
+
+def bad_sdgfsdg23245777():
```

Comment:
nit: but maybe choose some less distracting method names

Reply:
Ported from https://github.com/PyCQA/pydocstyle/blob/master/src/tests/test_cases/test.py#L353-L368

Reply:
Yes I don't think somebody smashing their keyboard is something we should port.